packages/website/tests/homepage.e2e.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect, Page, TestInfo } from '@playwright/test';
+import * as fs from 'fs';
+import { extname } from 'path';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/');
```

Comment:
Looks like tests are failing. Interesting that CI Job was not affected by it

Reply:
I forked where this was at into another branch and [tests pass](https://github.com/web3-storage/web3.storage/runs/5889791211?check_suite_focus=true).

In what way did it look like the tests are failing?
This was in orig PR:
> There are still [failish](https://github.com/web3-storage/web3.storage/actions/runs/2112017515) annotations on the checks page, but I think they are from older runs? the [newest runs are green](https://github.com/web3-storage/web3.storage/actions/workflows/website.yml)

Reply:
I was looking the logs of the workflow job: https://github.com/web3-storage/web3.storage/runs/5889916913?check_suite_focus=true#step:9:142

It is weird because it is green, but per the logs it keeps failing. Any ideas?

It looks like beforeEach always fails https://github.com/web3-storage/web3.storage/runs/5889916913?check_suite_focus=true#step:9:178

Reply:
@vasco-santos The only thing I can think of is that maybe playwright didn't consider that there were any test failures because the failure was in `beforeEach`, none of the tests were actually run?

I tested my latest changes [here](https://github.com/web3-storage/web3.storage/runs/5977853422?check_suite_focus=true) to try to ensure that failed tests also fail the CI

Reply:
>  that there were any test failures because the failure was in beforeEach, none of the tests were actually run?

Probably the case. Mind opening an issue in playwright test?

Reply:
I'm not able to validate that theory with a reduced test case: https://github.com/gobengo/test-playwright/runs/5994068552?check_suite_focus=true

The error [we saw](https://github.com/web3-storage/web3.storage/runs/5889916913?check_suite_focus=true#step:9:143) involved an `NS_ERROR_ABORT`, which I think indicates an aborted request in firefox, but that could be for any number of reasons?
https://github.com/microsoft/playwright/search?q=NS_ERROR_ABORT&type=issues